website/content/gateway/api_reference/endpoint_summary/endpoint_summary.md

```diff
@@ -0,0 +1,27 @@
+---
+title: "endpoint_summary"
+date: 2022-04-13T12:37:22+02:00
+anchor: "endpoint_summary"
+weight: 215
+---
+## Endpoint summary
+
+```shell
+# authorizations
+https://gateway.clearhaus.com/authorizations
+
+# captures
+https://gateway.clearhaus.com/authorizations/:id/captures
+
+# voids
+https://gateway.clearhaus.com/authorizations/:id/voids
+
```

Comment:
It's weird to have `voids` here, i.e. between `captures` and `refunds`.

The order in in the API Reference is to have `voids` between `refunds` and `credits`.

I know this is out of scope for this PR, though. On the other hand I wouldn't mind if we "corrected" it.